packages/ringcentral-widgets/components/ConferenceParticipantPanel/index.js

```diff
@@ -50,7 +52,10 @@ class ParticipantsContainer extends Component {
       && !nextProps.participants.find(
         participant => participant.id === this.state.detail.id
```

Comment:
split 'onCancel' and 'onClose' so we don't need to duplicate the `this.setState`